modules/oe_whitelabel_helper/oe_whitelabel_helper.module

```diff
@@ -84,3 +84,15 @@ function oe_whitelabel_helper_field_formatter_settings_summary_alter(array &$sum
     ]);
   }
 }
+
+/**
+ * Implements hook_requirements_alter().
+ */
+function oe_whitelabel_helper_requirements_alter(array &$requirements): void {
+  $theme_handler = \Drupal::service('theme_handler');
+  $theme_path = $theme_handler->getTheme('oe_bootstrap_theme')->getPath();
+  $path = '/' . $theme_path . '/assets/js/slimselect.min.js';
+  if (file_exists($path)) {
+    unset($requirements['slim_select_library']);
```

Comment:
rather then doing this we should mimic the modules behaviour and set a success message
```
  $requirements['slim_select_library'] = [
    'title' => t('Slim Select library'),
    'severity' => REQUIREMENT_OK,
    'value' => t('Library available at :path.', [
      ':path' => $path,
    ]),
  ];
```